dev-infra-squad.json

```diff
@@ -33,7 +33,7 @@
     "packageRules": [
         {
             "matchPackagePatterns": [
-                ".*sonar-(ubuntu|windowsserver)-.*"
+                ".*sonar-(ubuntu|windowsserver|amazon)-.*"
```

Comment:
The label is going to be missing if we start using CentOS: `sonar-centos-7-minimal-x86 2023-09-18T07-00-09.085193Z` for example. What do you think about adding it now?

Reply:
good idea, let's use already anticipate. I'm changing the PR